samples/unit-tests/datalabels/multiple-labels/demo.js

```diff
@@ -194,11 +194,13 @@ QUnit.test('Multiple data labels general tests.', function (assert) {
         reversed: false
     });
 
+    /* Fails after crisp refactor, but it's not a clear bug
```

Comment:
Looks like a bug here: bar type series' `dataLabels` with `inside: true` are showing for any y range - https://jsfiddle.net/BlackLabel/7stgvowh/1/

Reply:
Fixed